src/Kestrel.Core/Internal/FrameConnectionContext.cs

```diff
@@ -14,7 +14,7 @@ public class FrameConnectionContext
         public long FrameConnectionId { get; set; }
```

Comment:
Is this still a `FrameConnectionContext` and a `FrameConnectionId` as they are Http specific; is this context?